src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,129 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+export const FindMovie = ({
+  query,
+  changeQuery,
+  findMovie,
+  addMovie,
+  setQuery,
+}) => {
+  const [movie, createMovie] = useState({});
```

Comment:
```suggestion
  const [movie, createMovie] = useState(null);
```
use null as default value